src/compiler/execution.jl

```diff
@@ -41,7 +41,7 @@ macro cuda(ex...)
         split_kwargs(kwargs,
                      [:dynamic, :launch],
                      [:minthreads, :maxthreads, :blocks_per_sm, :maxregs, :name],
-                     [:cooperative, :blocks, :threads, :config, :shmem, :stream])
+                     [:cooperative, :blocks, :threads, :config, :shmem, :stream, :rng_seed])
```

Comment:
Maybe seeding ought to be an operation you should do explicitly in the kernel, to avoid this extra argument. You'd then call seed from a thread to set the RNG state for that specific thread.